static/webpack.config.js

```diff
@@ -20,6 +20,10 @@ const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries");
 
 const config = {
   entry: {
+    scatter2: [
```

Comment:
let's not forget to rename these after as well.

Reply:
added a TODO